scrapy_poet/__init__.py

```diff
@@ -1,2 +1,4 @@
-from .middleware import InjectionMiddleware
-from .utils import callback_for
+from .middleware import InjectionMiddleware, DEFAULT_PROVIDERS
```

Comment:
Do we need to expose DEFAULT_PROVIDERS on a top level? What's the use case?

Reply:
Maybe is too much? The unique use case is to be able to know which are the default providers. So maybe better to remove from the top level?

Reply:
Yeah, users always can import those directly. Top-level shortcuts are there to simplify common tasks & suggest what's important in the API. 

I think users may be able to see default providers in logs, at spider startup - do you know if this happens?

Reply:
Removed here: https://github.com/scrapinghub/scrapy-poet/pull/41/commits/7ac3911a35db0572309d198bc5cbec17f61ff1af and improving logging here: https://github.com/scrapinghub/scrapy-poet/pull/41/commits/02f7b2b92a9ddea107e071815d20e58774a4ace1

Reply:
Scrapy logs settings at startup, I was hoping this covers logging of the providers

Reply:
It doesn't seem to be the case.